clients/js/package.json

```
@@ -11,34 +11,44 @@
   ],
   "type": "module",
   "main": "lib/index.js",
+  "module": "lib/index.js",
   "types": "lib/index.d.ts",
+  "exports": {
+    ".": "lib/index.js",
```

I got this error when I tried to import this module:
```
Error [ERR_INVALID_PACKAGE_TARGET]: Invalid "exports" main target "lib/index.js" defined in the package config /Users/jameszheng/Documents/Oasis/ts-deployment/node_modules/@oasisprotocol/sapphire-paratime/package.json imported from /Users/jameszheng/Documents/Oasis/ts-deployment/build/index.js; targets must start with "./"
```

Do we need
```suggestion
    ".": "./lib/index.js",
```